src/ConnectorPermissions.ts

```diff
@@ -15,15 +14,19 @@ export class ConnectorPermissions {
     this.permissionsContract = permissionsContract;
   }
   
-  static async init(shardName: string,
+  static async init(
+    chain: Chain,
+    shardName: string,
```

Comment:
what is chain and what is shardName? Add comments

Reply:
This was clarified in conversation and we agreed that enums are descriptive enough for code. This will be documented in the SDK documentation